ibis/backends/flink/registry.py

```diff
@@ -442,9 +494,16 @@ def _struct_field(translator, op):
         # Binary operations
         ops.Power: fixed_arity("power", 2),
         ops.FloorDivide: _floor_divide,
-        # Collection functions
+        # Collection operations
+        ops.ArrayColumn: _array_column,
```

Comment:
Can you reduce some of these to their `fixed_arity` versions?

For example, `array_remove` can be defined as

```
ops.ArrayRemove: fixed_arity("ARRAY_REMOVE", 2)
```

Reply:
Thanks, changed the following two:
```
        ops.ArrayRemove: fixed_arity("ARRAY_REMOVE", 2),
        ops.ArrayUnion: fixed_arity("ARRAY_UNION", 2),
```